templates/report.html

```diff
@@ -0,0 +1,319 @@
+            display: block;
+            position: fixed;
+            top: 175px;
+            user-select: initial;
+            max-width: 20vw;
+            margin-left: 15px;
+            opacity: 0;
+            transition: opacity 300ms;
+        }
+        #ui-toc-affix .nav > li {
+            width: 100%;
+        }
+        .dropup .dropdown-menu {
+            bottom: auto !important;
+        }
+        @media (min-width: 768px) and (max-width: 991px) {
+            header .navbar-collapse {
+                padding-left: 0;
+            }
+            
+            header .navbar-nav > li > a {
+                padding-top: 10px;
+                padding-bottom: 10px;
+            }
+        }
+        @media (max-width: 768px) {
+            header .navbar-collapse {
+                padding-left: 0;
+            }
+
+            header .navbar-nav .nav-link {
+                padding-left: 15px;
+            }
+
+        }
+    </style>
+    <div id="post-content">
+        <div class="cover"></div>
+        <div class="container">
+            <div class="row article">
+                <div class="col">
+                    {% block post %}
+                        <article class="article-content">
+                            {{ include_raw(template) }}
+                        </article>
+                    {% endblock %}
+                </div>
+            </div>
+        </div>
+        <div id="post-footer" class="container post-footer">
+            <div class="authorbox">
+                {% for author in authors%}
+                    <div class="authorbox__content">
+                        <h3>{{author.name}}</h3>
+                        <h5>
+                            {{author.title}}
+                            <a href="{{author.url}}" target="_blank" rel="noopener">
+                                {{author.handle}}
+                            </a>
+                        </h5>
+                        
+                        <img src="{{author.image}}" width="120px" height="120px" alt="{{author.name}}"/>
+                    </div>
+                {%endfor%}
+            </div>
+            <h5>Published on <a href="#permalink">{{date}}</a></h5>
+            <div class="newsletterbox">
+                <a href="https://tinyletter.com/Monadical">
+                    Subscribe to our newsletter
+                </a>
+            </div>
+        </div>
+        <center>
+            <br/><br/><br/>
+            <i style="font-size: 18px">Recent posts:</i>
+            <ul style="max-width: 400px; text-align: left; font-size: 16px">
+                <br/><br/>
+                {% for post in POSTS.values() %}
+                    {% if loop.index < 5 %}
+                        <li><a href="{{post.url}}">{{post.title}}</a></li>
+                    {% endif %}
+                {% endfor %}
+                <li>
+                    <a href="{{PAGES.blog.url}}">
+                        View more posts...
```

Comment:
same here, change to reports instead of blog

Reply:
similar reasoning as above too :)